From 7aad4f152380e1d2edffdc9a2eb15554f042fbad Mon Sep 17 00:00:00 2001
From: Nick Schermer <nick@xfce.org>
Date: Sun, 18 Oct 2009 22:47:32 +0000
Subject: Only sync devices with no volume.

If the devices is pollable and we set the sync option, this
will slowdown the transfer. I missed that when changing the
code.
---
diff --git a/exo-mount/exo-mount-hal.c b/exo-mount/exo-mount-hal.c
index bbf9ca3..d09d0db 100644
--- a/exo-mount/exo-mount-hal.c
+++ b/exo-mount/exo-mount-hal.c
@@ -741,7 +741,8 @@ exo_mount_hal_device_mount (ExoMountHalDevice *device,
         {
           option = NULL;
 
-          if (strcmp (device->fsoptions[m], "sync") == 0)
+          if (strcmp (device->fsoptions[m], "sync") == 0
+              && device->volume == NULL)
             {
               /* non-pollable drive... */
               option = g_strdup ("sync");
--
cgit v0.8.2.1
